model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/changes/EmbeddedCachesChangesPerformer.java

```diff
@@ -50,19 +50,19 @@ private void runOperationInCluster(K key, MergedUpdate<V> task,  SessionEntityWr
         switch (operation) {
             case REMOVE:
                 // Just remove it
-                CacheDecorators.skipCacheStoreIfRemoteCacheIsEnabled(cache)
+                cache.getAdvancedCache()
```

Comment:
nitpicking, you could add a new method: `CacheDecorator.ignoreReturnValue(cache)`.